React/Modules/RCTAlertManager.m

```diff
@@ -141,13 +149,31 @@ - (void)invalidate
                                  preferredStyle:UIAlertControllerStyleAlert];
 
     if (allowsTextInput) {
-      [alertController addTextFieldWithConfigurationHandler:^(UITextField *textField) {
-        textField.text = message;
-      }];
-    } else {
-      alertController.message = message;
+      if ([type isEqual:@"plain-text"]) {
+        [alertController addTextFieldWithConfigurationHandler:^(UITextField *textField) {
+          textField.secureTextEntry = NO;
+        }];
+      } else if ([type isEqual:@"secure"]) {
+          [alertController addTextFieldWithConfigurationHandler:^(UITextField *textField) {
+            textField.placeholder = @"Password";
+            textField.secureTextEntry = YES;
+          }];
+      } else if ([type isEqual:@"login-password"]) {
+        [alertController addTextFieldWithConfigurationHandler:^(UITextField *textField) {
+          textField.placeholder = @"Login";
```

Comment:
Does iOS provide any pre-localized constants that we can use for these fields? Hard-coding the English text doesn't seem like a very good solution in the long term.